src/utils/Inspect.ts

```diff
@@ -0,0 +1,32 @@
+export {};
+
+declare global {
+  interface Window {
+    __insp: Array<Array<string | number>>,
+    __inspld: Number
+  }
+}
+
+const InspectLet = () => {
+    window.__insp = window.__insp || [];
+    window.__insp.push(['wid', 812310448]);
```

Comment:
What is this key (812310448) where someone can get it from?

Reply:
This is the wid number we can get from Inspectlet Dashboard, I will pass it as const on top of this file.